src/feature/wines/components/card/WineCard.tsx

```
@@ -0,0 +1,87 @@
+  ),
+  content: cva('flex gap-4 md:gap-6'),
+  footer: cva(
+    'border-primary border-t-2 p-6 flex justify-center items-center md:px-24 md:pt-12',
+  ),
+};
+
+const WineCard = ({ wine }: { wine: WineListResponse['list'][number] }) => {
+  const { name, region, image, price, avgRating, reviewCount, recentReview } =
+    wine;
+  return (
+    <Card className={wineCardVariants.card()} role='article'>
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<article>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)